Authenticator/Source/TokenList.swift

```diff
@@ -159,3 +159,35 @@ extension TokenList {
         return .ShowSuccessMessage("Copied")
     }
 }
+
+extension TokenList.Action: Equatable {}
+func == (lhs: TokenList.Action, rhs: TokenList.Action) -> Bool {
+    switch (lhs, rhs) {
+    case (.BeginAddToken, .BeginAddToken):
+        return true
+    case let (.EditPersistentToken(l), .EditPersistentToken(r)):
```

Comment:
I prefer to avoid catch-all cases where possible, though unfortunately I haven't found a good way to implement enum equality that doesn't require one. I think I prefer the structure where each `case` recognizes when actions are the same type, and then the following line compares for equality in the context of that action.